connaisseur/alert.py

```diff
@@ -53,31 +54,66 @@ class AlertReceiverAuthentication:
     Class to store authentication information for securely sending events to the alert receiver.
     """
 
-    class AlertReceiverBasicAuthentication:
+    authentication_config: dict = None
+    authentication_scheme: str = None
+
+    class AlertReceiverAuthenticationInterface:
+        def __init__(self, alert_receiver_config: dict, authentication_key: str):
+            self.authentication_config = alert_receiver_config.get(authentication_key)
```

Comment:
`authentication_key` sounds like a credential

Reply:
Yes, I agree. It will be called `authentication_config_key`.